### ISSUE TITLE
Add no-select css

### ISSUE THREAD

**Issue by bencooper222** [VandyHacks/VH6-Placeholder#10]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/9143563/61096190-8ab19480-a40b-11e9-908c-c55c2215962f.png">

Assigning for visibility.

**Comment by k-zhai** [comment]
Should be fixed.

**Comment by bencooper222** [comment]
Yeah I thought I saw a PR for it. This happened on the deploy link in the Slab notes - is that link out of date or something?

**Comment by geomaster453** [comment]
yeah i think Kevin just merged some changes a few minutes ago.

**Comment by timothy-liang** [comment]
Still broken on Firefox I think.

**Comment by cktang88** [comment]
what's the most updated link? @k-zhai
